### PR TITLE
Intro cinematic

### PR DESCRIPTION
**Pull request by cmeese549**
nice

### CODE REVIEW

**Comment by cmeese549** [comment]
oh also this fixes player ship prefab

File: Assets/Scripts/Weapons/Gun.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ void Update()
             heat -= Time.deltaTime * heatDecayRate;
         }
         
-        if(firing)
+        if(firing && realteam == Teams.playerTeam)
```
Comment (jakemcleman) [inline Assets/Scripts/Weapons/Gun.cs]: I'm not sure I like this way of disabling the overheat for enemies. Can we have a public boolean for `overheatEnabled` instead?

File: Assets/Scripts/Weapons/Gun.cs [hunk 2]
```diff
@@ -130,7 +130,11 @@ public bool Fire(IWieldable firer)
         fireClock = 0;
         float heatMod = heat;
         float maxHeatMod = maxHeat;
-        float pitchBend = heatMod / maxHeatMod + 0.3f;
+        float pitchBend = Random.Range(.1f, 1f);
+        if (realteam == Teams.playerTeam)
```
Comment (jakemcleman) [inline Assets/Scripts/Weapons/Gun.cs]: Wouldn't this make problems if there are friendly AI?

File: Assets/Scripts/Whale/WhaleAI.cs [hunk 3]
```diff
@@ -9,7 +9,7 @@ public class WhaleAI : MonoBehaviour
     private Transform[] RailPoints;
     private AIShip flightController;
 
-    private int curPoint = 0;
+    public int curPoint = 0;
```
Comment (jakemcleman) [inline Assets/Scripts/Whale/WhaleAI.cs]: I think I'd prefer the pattern of keeping this private but adding a public member for `InitialPoint`

**Review by jakemcleman — CHANGES_REQUESTED** [review]

**Comment by cmeese549** [comment]
Is did

**Review by jakemcleman — APPROVED** [review]

**Pull request merged**